tests/testdata/regionalisation_data/GetResgionalizationData.py

```diff
@@ -0,0 +1,164 @@
+                RunSuccessLandUse[i-1]=1
+                
+                # Save file
+                zippedList =  list(zip(ID, StationID, Area, Latitude,Longitude,Gravelius,	Perimeter,	RunSuccessShape,Elevation,	Slope,Aspect,RunSuccessTerrain,Forest,Grass,	Wetland,Water,Urban,Shrubs,Crops,	SnowIce,RunSuccessLandUse)) 
+                frameList = pd.DataFrame(data=zippedList, columns=['ID', 'StationID', 'area', 'latitude','longitude','gravelius','perimeter','RunSuccessShape','elevation','slope','aspect','RunSuccessTerrain','forest','grass','wetland','water','urban','shrubs','crops','snowIce','RunSuccessLandUse'])
+                frameList.to_csv('gauged_catchment_properties.csv')
+            except:
+                print('Failed, restarting iter ' + str(attempt))  
+            else:
+                break
+            
+    # TERRAIN ANALYSIS        
+    if RunSuccessTerrain[i-1]==0:
+        for attempt in range(1):
+            try:
+                
+                print('Doing Basin: ' + str(i))
+                shape2='/home/ets/SHP_ZIP/Basin_' + str(i) +'.zip'         
+                
+                resp=wps.terrain_analysis(shape=shape2, select_all_touching=True,projected_crs=3978)
+                while resp.isNotComplete():
+                    time.sleep(1)
+                
+                [properties]=resp.get(asobj=True)
+                Elevation[i-1]=properties[0]['elevation']
+                Slope[i-1]=properties[0]['slope']
+                Aspect[i-1]=properties[0]['aspect']
+                RunSuccessTerrain[i-1]=1
+                
+                # Save File
+                zippedList =  list(zip(ID, StationID, Area, Latitude,Longitude,Gravelius,	Perimeter,	RunSuccessShape,Elevation,	Slope,Aspect,RunSuccessTerrain,Forest,Grass,	Wetland,Water,Urban,Shrubs,Crops,	SnowIce,RunSuccessLandUse)) 
+                frameList = pd.DataFrame(data=zippedList, columns=['ID', 'StationID', 'area', 'latitude','longitude','gravelius','perimeter','RunSuccessShape','elevation','slope','aspect','RunSuccessTerrain','forest','grass','wetland','water','urban','shrubs','crops','snowIce','RunSuccessLandUse'])
+                frameList.to_csv('gauged_catchment_properties.csv')
+                
+            except:
+                print('Failed, restarting iter ' + str(attempt))  
+            else:
+                break
+      
+        
+    # SHAPE ANALYSIS
+    if RunSuccessShape[i-1]==0:
+        for attempt in range(1):
+            try:
+                
+                print('Doing Basin: ' + str(i))
+                shape2='/home/ets/SHP_ZIP/Basin_' + str(i) +'.zip'         
+                
+                resp = wps.shape_properties(shape=shape2, projected_crs=32198)
+                
+                while resp.isNotComplete():
+                    time.sleep(1)
+                
+                [properties]=resp.get(asobj=True)
+                Area[i-1]=properties[0]['area']/1000000.0
```

Comment:
I suggest we change the returned values so you can use the properties dictionary directly, without transformation.